docs/root/configuration/other_features/dlb.rst

```diff
@@ -0,0 +1,42 @@
+.. _config_connection_balance_dlb:
+
+DLB Connection Balancer
+=======================
+
+* :ref:`v3 API reference <envoy_v3_api_msg_extensions.dlb.v3alpha.Dlb>`
+
+
+This connection balancer extension provides Envoy with low latency networking by integrating with `Intel DLB <https://networkbuilders.intel.com/solutionslibrary/queue-management-and-load-balancing-on-intel-architecture>`_ through the libdlb library.
+
+The DLB connection balancer is only included in :ref:`contrib images <install_contrib>`
+
+Example configuration
+---------------------
+
+.. code-block:: yaml
+
+  static_resources:
+    listeners:
+      connection_balance_config:
+        extend_balance:
+          name: envoy.network.connection_balance.dlb
+          typed_config:
+            "@type": type.googleapis.com/envoy.extensions.dlb.v3alpha.Dlb
+
+How it works
+------------
+
+If enabled, the DLB connection balancer will
+- attache DLB hardware
```

Comment:
this list is collapsing - i think it needs a space above

Reply:
```suggestion
- attach DLB hardware
```

Reply:
Sure.